tests/test_parser.py

```diff
@@ -246,14 +246,18 @@ def test_bad_connection_units(self):
 
         assert match in str(dim_error.value)
 
-    def test_new_base_units(self):
-        """Tests unit checking in a model that defines a new base unit."""
+    def test_algebraic_and_new_base_units(self):
```

Comment:
Can we split this in two tests? These are two entirely unrelated things

Reply:
We could, but keep the same CellML file?

Reply:
sure